cogs/LogReading.py

```diff
@@ -0,0 +1,73 @@
+import discord
+import util.JsonHandler
+import os
+from discord.ext import commands
+import sys
+
+class LogReading(commands.Cog):
+
+    intents = discord.Intents.default()
+    intents.messages = True
+    intents.message_content = True
+
+
+    @commands.hybrid_command()
+    async def ping(self, ctx, arg):
+        await ctx.send("pong")
```

Comment:
also we should remove this, it was just added by Liz originally to test if the bot was online properly